packages/react/src/component-tests/IcPageHeader/IcPageHeader.cy.tsx

```diff
@@ -0,0 +1,282 @@
+/* eslint-disable react/jsx-no-bind */
+/// <reference types="Cypress" />
+
+import React from "react";
+import {
+  IcBreadcrumb,
+  IcBreadcrumbGroup,
+  IcButton,
+  IcChip,
+  IcNavigationItem,
+  IcPageHeader,
+  IcStep,
+  IcStepper,
+  IcTextField,
+} from "../../components";
+import { mount } from "cypress/react";
+import { BE_VISIBLE, HAVE_LENGTH } from "../utils/constants";
+import {
+  DesktopScrollablePageVariant,
+  ScrollStickyPageHeader,
+} from "./IcPageHeaderTestData";
+
+const DEFAULT_TEST_THRESHOLD = 0.05;
+
+describe("IcPageHeader e2e, A11y and visual regression tests", () => {
+  beforeEach(() => {
+    cy.injectAxe();
+  });
+
+  afterEach(() => {
+    cy.task("generateReport");
+  });
+
+  it("IcPageHeader", () => {
+    mount(
+      <IcPageHeader
+        heading="Latte recipe"
+        subheading="A Latte is a popular Italian coffee, made with espresso, steamed milk and a thin layer of foam. "
+        reverseOrder
+      >
+        <IcChip slot="heading-adornment" label="BETA" size="large" />
+      </IcPageHeader>
+    );
+    cy.checkHydrated("ic-page-header");
+    cy.checkA11yWithWait();
+    cy.compareSnapshot({
+      name: "default",
+      testThreshold: DEFAULT_TEST_THRESHOLD + 0.05,
+    });
+  });
+
+  it("should stay at the top of the page if the sticky variant is used", () => {
+    mount(<ScrollStickyPageHeader />);
+    cy.viewport(1050, 450);
+    cy.checkA11yWithWait();
+    cy.scrollTo("bottom").checkHydrated("ic-button#end");
+    cy.checkA11yWithWait();
+    cy.get("ic-page-header").invoke("css", "position", "absolute");
```

Comment:
Saw this on the Cypress site but I do not think this is what we want because it changes the position from `sticky`.